pkg/controller/pkg/revision/reconciler.go

```diff
@@ -52,11 +52,10 @@ const (
 
 	errInitParserBackend = "cannot initialize parser backend"
 	errParseLogs         = "cannot parse pod logs"
+	errOneMeta           = "cannot install package with multiple meta types"
```

Comment:
Nit: `errNotOneMeta` or `errMultiMeta`? This reads like the error is because there is one meta, not because there must be one meta.